src/style.css

```diff
@@ -0,0 +1,25 @@
+body {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 13px;
+}
+
+h1 {
+  margin: 0;
+  font-size: 17px;
+  font-weight: bold;
```

Comment:
do you really need this?

Reply:
test shows errors, when change. What shoud I do?

Reply:
just removed  -   font-weight: bold;